coap-gateway/service/request.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func checkReq(request CoapSignInReq) error {
+	if request.UserID == "" {
+		return status.Errorf(codes.InvalidArgument, "invalid UserId")
+	}
+	if request.AccessToken == "" {
+		return status.Errorf(codes.InvalidArgument, "invalid AccessToken")
+	}
+	return nil
+}
```

Comment:
why it is a separate file -> it is below to sign I think.

Reply:
Will be used by updated `refresh_token` and `sign_off` implementations which currently use a different `checkReq` function, but will be switched when new implementation is done, so moving this to separate file makes sense